openmdao/recorders/recording_manager.py

```diff
@@ -87,6 +87,10 @@ def startup(self, root):
                 self._vars_to_record['unames'].update(unames)
                 self._vars_to_record['rnames'].update(rnames)
 
+    def close(self):
```

Comment:
Missing docstring. Also, I re-organized and cleaned up this class in my pull request.